cirras-underwriting-api/cirras-underwriting-api-rest-endpoints/src/test/java/ca/bc/gov/mal/cirras/underwriting/api/rest/v1/endpoints/VerifiedYieldContractEndpointTest.java

```diff
@@ -266,12 +304,12 @@ public void testGetVerifiedYieldContractCommodities() throws CirrasUnderwritingS
 
 		checkVerifiedYieldContractCommodities(expectedVycc, verifiedYldContract.getVerifiedYieldContractCommodities());
 		
+		checkVerifiedYieldAmendments(expectedVya, verifiedYldContract.getVerifiedYieldAmendments());
```

Comment:
Need a test for VerifiedYieldContract.fields and AnnualFieldRsrc.verifiableCommodities.